src/data/reducers/emailTemplate.js

```diff
@@ -68,6 +78,33 @@ const emailTemplate = (state = initialState, action) => {
         error: null,
         ...transformTemplates(action.payload.data, initialState),
       };
+    case ALL_EMAIL_TEMPLATE_SUCCESS:
+      return {
+        ...state,
+        loading: false,
+        error: null,
+        allTemplates: action.payload.data,
```

Comment:
I think it would be great if we can use the existing `assign`, `remind` and `revoke` in store and make them arrays of related templates instead of introducing a new field(`allTemplates`) store